packages/web/app/package.json

```diff
@@ -107,7 +101,9 @@
     "@welldone-software/why-did-you-render": "7.0.1",
     "autoprefixer": "10.4.13",
     "postcss": "8.4.21",
-    "rimraf": "4.1.2"
+    "rimraf": "4.1.2",
+    "tailwindcss": "3.2.6",
+    "tailwindcss-radix": "2.7.0"
```

Comment:
since now we no longer extend chakra theme we can put tailwindcss deps to devdeps